code/src/components/NameQuestion.js

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+
+export const NameQuestion = ({ name, onNameChange }) => {
+  
+  return (
+    
+    <label>
+      <h2> What is your name?</h2>
+      <div className="name-box">
+      <input
+        type="text"
+        maxLength="30"
+        minLength="3"
```

Comment:
Learned something new here, thanks! :D Didn’t know that we could put max and minLenght here!